var/spack/repos/builtin/packages/singularityce/package.py

```diff
@@ -24,9 +24,7 @@ class SingularityBase(MakefilePackage):
     depends_on("git", when="@develop")  # mconfig uses it for version info
     depends_on("shadow", type="run", when="@3.3:")
     depends_on("cryptsetup", type=("build", "run"), when="@3.4:")
-    # BlueBrain: build failed due to missing dependency
-    # It's not in repo-patches because then the build fails due to missing spack_perms_fix.sh
-    depends_on('glib@2', type=('build', 'run'))
```

Comment:
This dependency is no longer needed?

Reply:
Yes. This was only needed to install `conmon` which is now coming as an external dependency